AsyncDisplayKit/ASControlNode.mm

```diff
@@ -77,23 +77,35 @@ - (id)init
 {
   if (!(self = [super init]))
     return nil;
-
+  
   _enabled = YES;
-
+  
   // As we have no targets yet, we start off with user interaction off. When a target is added, it'll get turned back on.
   self.userInteractionEnabled = NO;
+  
   return self;
 }
 
+- (void)didLoad
+{
+#if TARGET_OS_TV
+  //On tvOS all control views, such as buttons, interact with the focus system even if they don't have a target set on them. Here we add our own internal tap gesture to handle this behaviour.
+  self.userInteractionEnabled = YES;
+  UITapGestureRecognizer *tapGestureRec = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(pressDown)];
+  tapGestureRec.allowedPressTypes = @[@(UIPressTypeSelect)];
+  [self.view addGestureRecognizer:tapGestureRec];
+#endif
+}
+
 #pragma mark - ASDisplayNode Overrides
 - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
 {
   // If we're not interested in touches, we have nothing to do.
   if (!self.enabled)
     return;
-
+  
```

Comment:
@appleguy Huh, I was under the assumption Xcode would respect the current spacing when using Ctrl+I. Will revert those space changes.